test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1964,35 +1964,6 @@ def forward(self, x):
         x = torch.randn(2, 3, 4)
         self.run_test(RandLike(), x)
 
-    def _interpolate(self, x, mode, use_size, is_upsample, align_corners=False):
-        class MyModel(torch.nn.Module):
-            def forward(self, x):
-                scale = 2.3 if is_upsample else 0.5
-                if len(x.size()) == 3:
-                    scale_array = 2.3
-                if len(x.size()) == 4:
-                    scale_array = [2.3, 5.1]
-                if len(x.size()) == 5:
-                    scale_array = [3.3, 2.3, 5.1]
-                if use_size:
```

Comment:
As you delete this model, the question is: Is this `use_size=True` path covered in the UT?

Reply:
Yes, this part is a old usage which is used for exporting tracing test. Now the _interpolate_script default covered both scripting and tracing test, it's also covered use_size=True.